docs-chef-io/content/workstation/getting_started.md

```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
-specific command shell. For example, if Bash were the command shell and
-the configuration file were named `bash_profile`, the command would look
-something like the following:
+{{< /foundation_tabs_panels >}}
+<!---* End Panels --->
+### Add Ruby as an Environment Variable
 
-``` bash
-echo 'export PATH="/opt/chef-workstation/embedded/bin:$PATH"' >> ~/.bash_profile && source ~/.bash_profile
-```
+Chef Workstation includes a stable version of Ruby as part of its installer.
+To add the Chef Ruby as an environment variable, add the path to the Chef Ruby to the `$PATH` environment variable and save it in the configuration file for the command shell.
 
-{{< warning >}}
+<!---* Tabs Section--->
+{{< foundation_tabs tabs-id="tabs-panel-container" >}}
+{{< foundation_tab active="true" panel-link="ruby-env-macOS-panel" tab-text="macOS/Linux">}}
+{{< foundation_tab panel-link="ruby-env-windows-panel" tab-text="Windows" >}}
+{{< /foundation_tabs >}}
+<!---* End Tabs --->
 
-On Microsoft Windows, `C:/opscode/Chef Workstation/bin` must be before
-`C:/opscode/Chef Workstation/embedded/bin` in the `PATH`.
+<!---* Panels Section --->
+{{< foundation_tabs_panels tabs-id="tabs-panel-container" >}}
+{{< foundation_tabs_panel active="true" panel-id="ruby-env-macOS-panel" >}}
 
-{{< /warning >}}
+  On Linux systems with Bash as the default shell, your configuration file is called `.bashrc`. The command for adding the Chef Ruby path to your `.bashrc` file is:
 
-## Create the Chef repository
+  ``` bash
+  echo 'export PATH="/opt/chef-workstation/embedded/bin:$PATH"' >> ~/.bashrc && source ~/.bashrc
+  ```
 
-Use [the chef generate repo]({{< relref "ctl_chef.md#chef-generate-repo" >}}) to
-create the Chef repository. For example, to create a repository called
-`chef-repo`:
+  On macOS systems with Zsh as the default shell, If you use Zsh as your command shell, your configuration file is called `.zshrc`. The command for adding the Chef Ruby path to your `.zshrc` file is:
 
-``` bash
-chef generate repo chef-repo
-```
+  ``` bash
+  echo 'export PATH="/opt/chef-workstation/embedded/bin:$PATH"' >> ~/.zshrc && source ~/.zshrc
+  ```
 
-### Install a Code Editor
-
-A good visual code editor is not a requirement for working with Chef
-Infra, but a good code editor can save you time. A code editor should
-support the following: themes, plugins, snippets, syntax Ruby code
-coloring/highlighting, multiple cursors, a tree view of the entire
-folder/repository you are working with, and a Git integration.
-
-These are a few common editors:
+{{< /foundation_tabs_panel >}}
+{{< foundation_tabs_panel panel-id="ruby-env-windows-panel" >}}
+{{< warning >}}
 
-- [Visual Studio Code (free/open source)](http://code.visualstudio.com)
-- [GitHub Atom - (free/open source)](http://atom.io)
+  On Microsoft Windows, `C:/opscode/Chef Workstation/bin` must be before `C:/opscode/Chef Workstation/embedded/bin` in the `PATH`.
```

Comment:
We probably want to describe how to add to the path via either the GUI or via PowerShell here before we warn them about the order.